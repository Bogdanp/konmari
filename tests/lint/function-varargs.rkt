#lang racket/base

(define (f x)
  (void))

(define (f . xs)
  (void))

(define (g a b)
  (void))

(define (h a b . xs)
  (void))

(define (i x [y 1] #:z z #:w [w z] . args)
  (cons w args))
