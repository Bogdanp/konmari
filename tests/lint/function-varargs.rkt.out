/function-varargs.rkt:3:12:warning:identifier 'x' is never used
/function-varargs.rkt:6:10:error:identifier 'f' is already defined
/function-varargs.rkt:6:14:warning:identifier 'xs' is never used
/function-varargs.rkt:9:14:warning:identifier 'b' is never used
/function-varargs.rkt:9:12:warning:identifier 'a' is never used
/function-varargs.rkt:12:14:warning:identifier 'b' is never used
/function-varargs.rkt:12:18:warning:identifier 'xs' is never used
/function-varargs.rkt:12:12:warning:identifier 'a' is never used
/function-varargs.rkt:15:15:warning:identifier 'y' is never used
/function-varargs.rkt:15:12:warning:identifier 'x' is never used
/function-varargs.rkt:9:10:warning:identifier 'g' is never used
/function-varargs.rkt:3:10:warning:identifier 'f' is never used
/function-varargs.rkt:12:10:warning:identifier 'h' is never used
/function-varargs.rkt:15:10:warning:identifier 'i' is never used
